docs/source/getting-started/installation.rst

```diff
@@ -15,9 +15,12 @@ Install SkyPilot using pip:
   $ # pip install "skypilot[gcp]"
   $ # pip install "skypilot[azure]"
   $ # pip install "skypilot[lambda]"
+  $ # pip install "skypilot[ibm]"
+  $ # pip install "skypilot[scp]"
+  $ # pip install "skypilot[oci]"
   $ # pip install "skypilot[all]"
 
-SkyPilot currently supports seven cloud providers: AWS, GCP, Azure, Lambda Cloud, IBM, SCP, and Cloudflare (for R2 object store).
+SkyPilot currently supports seven cloud providers: AWS, GCP, Azure, Lambda Cloud, IBM, SCP, OCI, and Cloudflare (for R2 object store).
```

Comment:
```suggestion
SkyPilot currently supports these cloud providers: AWS, GCP, Azure, Lambda Cloud, IBM, SCP, OCI, and Cloudflare (for R2 object store).
```
So that we don't have to update it